hi-ml-histopathology/src/histopathology/utils/output_utils.py

```diff
@@ -72,6 +72,8 @@ def normalize_dict_for_df(dict_old: Dict[ResultsKey, Any]) -> Dict[str, Any]:
                 value = value.squeeze(0).cpu().numpy()
                 if value.ndim == 0:
                     value = np.full(bag_size, fill_value=value)
+            if isinstance(value, List) and isinstance(value[0], torch.Tensor):
```

Comment:
This function could do with a lot more documentation. A docstring would be great. And each of the `if` branches should have a clear description which case we are handling here, and where those cases arise.